pages/index.tsx

```diff
@@ -4,33 +4,35 @@ import {
   withAuthUser,
   withAuthUserTokenSSR,
   AuthAction,
+  AuthUserContext,
 } from "next-firebase-auth";
 import Header from "../components/Header";
 import DemoPageLinks from "../components/DemoPageLinks";
 import getAbsoluteURL from "../utils/getAbsoluteURL";
 
-const styles = {
-  content: {
-    padding: 32,
-  },
-  infoTextContainer: {
-    marginBottom: 32,
-  },
+const addDBRecord = async (AuthUser: AuthUserContext) => {
+  const token = await AuthUser.getIdToken();
+  const response = await fetch("/api/createRecord", {
+    method: "POST",
+    headers: {
+      Authorization: token || "unauthenticated",
+    },
+  });
+  const data = await response.json();
+  console.log(data);
 };
 
-const Demo = ({ favoriteColor }) => {
+const Index = () => {
```

Comment:
Moved this page to `/`. If an unauthenticated user visits it, they will be redirected to `/login`. This is where we'll put our app's main screen